cmake/functions.cmake

```diff
@@ -884,7 +887,7 @@ function(generate_lib_src_list LIB_SOURCES)
 endfunction()
 
 function(add_to_options_file DEFINITIONS OPTION_FILE)
-    list(REMOVE_DUPLICATES DEFINITIONS)
+    #list(REMOVE_DUPLICATES DEFINITIONS)
```

Comment:
I assume this was done because options.h for the configure flow isn't de-duplicated. I think we should keep this in CMake and add de-duplication for configure.

Reply:
Yeah, I will fix that with rebase. The configure de-duplication doesn't appear trivial.